forum/projects/serializers.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework import serializers
+from .models import Project
+
+
+class ProjectSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Project model
+    """
+    class Meta:
+        model = Project
+        fields = '__all__'
+    
+    def validate_budget(self, value):
+        if value < 0:
+            raise serializers.ValidationError("Budget can not be negative")
+        return value
+    

```

Comment:
May be it's not obligatory to declare methods `create` and `update` if you use `ModelSerializer` and if you don't use some custom logic in these methods.